src/services/outliningElementsCollector.ts

```diff
@@ -4,25 +4,13 @@ namespace ts.OutliningElementsCollector {
         const elements: OutliningSpan[] = [];
         const collapseText = "...";
 
-        function addOutliningSpan(hintSpanNode: Node, startElement: Node, endElement: Node, autoCollapse: boolean) {
+        function addOutliningSpan(hintSpanNode: Node, startElement: Node, endElement: Node, autoCollapse: boolean, fullStart: boolean) {
             if (hintSpanNode && startElement && endElement) {
                 const span: OutliningSpan = {
-                    textSpan: createTextSpanFromBounds(startElement.pos, endElement.end),
-                    hintSpan: createTextSpanFromNode(hintSpanNode, sourceFile),
-                    bannerText: collapseText,
-                    autoCollapse,
-                };
-                elements.push(span);
-            }
-        }
-
-        function addOutliningForObjectLiteralsInArray(startElement: Node, endElement: Node, autoCollapse: boolean) {
-            if (startElement && endElement) {
-                const span: OutliningSpan = {
-                    textSpan: createTextSpanFromBounds(startElement.getStart(), endElement.end),
+                    textSpan: createTextSpanFromBounds(fullStart ? startElement.pos : startElement.getStart(), endElement.end),
                     hintSpan: createTextSpanFromBounds(startElement.getStart(), endElement.end),

```

Comment:
I don't think it should apply to the hint span, since it has always used the actual start (as opposed to the full start, which the textSpan previously always used).